File: parlio-tweede-kamer-graphql-api/src/test/java/nl/parlio/api/tweedekamer/JavaVersionTest.java
package nl.parlio.api.tweedekamer;

import static org.assertj.core.api.Assertions.assertThat;
import org.junit.jupiter.api.Test;

public class JavaVersionTest {

  @Test
  void isOnJava17() {
    int version = Runtime.version().feature();
    assertThat(version).isEqualTo(17);
  }
}

File: parlio-tweede-kamer-graphql-api/src/test/kotlin/nl/parlio/api/tweedekamer/JavaVersionTest.kt
package nl.parlio.api.tweedekamer

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class JavaVersionTest {

    @Test
    fun isOnJava17() {
        val version = Runtime.version().feature()
        assertThat(version).isEqualTo(17)
    }
}